ddtrace/contrib/grpc/aio_client_interceptor.py

```diff
@@ -52,11 +53,15 @@ def _handle_add_callback(call, callback):
         callback(call)
 
 
-def _done_callback(span, code, details):
-    # type: (Span, grpc.StatusCode, str) -> Callable[[aio.Call], None]
+def _done_callback(span, code=None, details=None):
     def func(call):
-        # type: (aio.Call) -> None
+        nonlocal code, details
```

Comment:
if we don't pass these in, we can drop this

Reply:
ok